### ISSUE TITLE
Django 2.0 issue

### ISSUE THREAD

**Issue by egorsmkv** [grantmcconnaughey/django-avatar#164]
Hello. I have an exception after connecting this library to my project:

```
...

  File "/var/virtualfish/***.**/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 673, in exec_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "/var/virtualfish/***.**/lib/python3.5/site-packages/avatar/models.py", line 90, in <module>
    class Avatar(models.Model):
  File "/var/virtualfish/***.**/lib/python3.5/site-packages/avatar/models.py", line 93, in Avatar
    verbose_name=_("user"),
TypeError: __init__() missing 1 required positional argument: 'on_delete'

```

**Comment by anuj9196** [comment]
#163

**Comment by grantmcconnaughey** [comment]
Version 4.1.0 has been released on PyPI. That should fix the Django 2.0 issues. https://pypi.python.org/pypi/django-avatar/4.1.0

**Comment by egorsmkv** [comment]
Thanks @anuj9196 @grantmcconnaughey
